### ISSUE TITLE
Videos upside down

### ISSUE THREAD

**Issue by doubleteapot** [KyleKun/one_second_diary#133]
There is the odd video that is imported upside down even though it shows up the right way in Google photos. I wonder if it is when I hold my phone upside down from the normal way to record the video but my phone automatically flips the video 180 degrees?? From memory I tried rotating the video and saving a new copy last time this happened, but it stayed upside down once imported.

**Comment by doubleteapot** [comment]
Yes it definitely still stays upside down when I rotate and save as a new video. I tried it in both the Google photos editor and the standard Android video viewer/editor. No matter which way up I rotate the video before saving, it still appears in the original filming orientation eg upside down

**Comment by koktavy** [comment]
I have this same issue, even when rotating the video a few degrees and saving as a new clip. The video is just upside down. Pixel 7 for reference.

**Comment by doubleteapot** [comment]
This happened to me again. I definitely held my camera up the opposite way to what I normally do this time, so that is definitely the cause. And I gather most apps would be able to flip the video 180 degrees automatically, but this app doesn't so it uses the meta data which says the video is upside down?

**Comment by alexanderadam** [comment]
Is the upside down issue maybe also related to issue #73?
Maybe it's really just the visualisation in both cases?

But Flutter generally seem to have some bug potentials there.
So there was [this unsolved issue about "Landscape videos are displayed upside down"](https://github.com/flutter/flutter/issues/62400) and also ["Videos recorded with an orientation of 180° (`landscapeRight`) are reversed"](https://github.com/flutter/flutter/issues/60327) and funnily enough also ["[video_player] The video is upside down when it is recorded at `landscape left`"](https://github.com/flutter/flutter/issues/137047).

**Comment by koktavy** [comment]
While the preview is still upside down, the saved clip is correctly flipped.

**Comment by TerenceAbigail** [comment]
This should be fixed after updating dependency in https://github.com/KyleKun/one_second_diary/pull/147

**Comment by doubleteapot** [comment]
Ah thanks koktavy, yes although the preview is still upside down, the saved clip is the right way up, so that's good at least. Still confusing though.
